app/src/org/commcare/fragments/FileServerFragment.java

```diff
@@ -138,21 +131,21 @@ protected String doInBackground(Void... params) {
                     Log.d(TAG, "server: copying files " + f.toString());
                     InputStream inputstream = client.getInputStream();
                     CommCareWiFiDirectActivity.copyFile(inputstream, new FileOutputStream(f));
-                    serverSocket.close();
                     publishProgress("copied files: " + f.getAbsolutePath(), f.getAbsolutePath());
                     publishProgress("File Server Resetting", null);
                     return f.getAbsolutePath();
 
                 } catch (IOException e) {
-                    Log.e(TAG, e.getMessage());
-                    final File f = new File(finalFileName);
-                    if (f.exists()) {
-                        FileUtil.deleteFileOrDir(f);
-                    }
-                    publishProgress("File Server crashed with an IO Exception: " + e.getMessage());
+                    Logger.log(TAG, e.getMessage());
```

Comment:
Seems like this it would be more helpful to pass the same message to `Logger.log()` as you are to `publishProgress()`

Reply:
good call